packages/build/src/plugins_core/secrets_scanning/index.ts

```diff
@@ -103,6 +121,7 @@ const coreStep: CoreStepFunction = async function ({
         enhancedSecretsScanMatchesCount: enhancedSecretMatches.length,
         secretsFilesCount: scanResults.scannedFilesCount,
         keysToSearchFor,
+        enhancedPrefixMatches: enhancedSecretMatches.length ? enhancedSecretMatches.map((match) => match.key) : [],
```

Comment:
Can we add:
```
enhancedScanning: enhancedSecretScan && enhancedScanningEnabledInEnv
```
here to be able to find spans with it enabled for some kind of monitoring to easily know wether enhanced scanning happened for the build?